.github/workflows/release.yml

```diff
@@ -0,0 +1,46 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
the main branch appears to be `master`, this change needs to be made in a few places